src/client/styles/main.scss

```diff
@@ -19,16 +19,11 @@ Import style rules from NYPL React module components
 @import "~@nypl/dgx-header-component/dist/styles/main";
 @import "~@nypl/dgx-react-footer/dist/styles/styles";
 
-@import "components/Account.scss";
-@import "components/Advanced.scss";
 @import "components/BibPage.scss";
-@import "components/Facets.scss";
 @import "components/Feedback.scss";
 @import "components/HoldPage.scss";
 @import "components/Home.scss";
-@import "components/Tabs.scss";
 @import "components/_forms.v2.scss";
```

Comment:
i've removed `_forms.v2.scss` from the TK you can probably remove this from your import list